nwp/__init__.py

```diff
@@ -7,6 +7,6 @@
 
 defs = Definitions(
     assets=all_assets,
-    jobs=jobs.asset_jobs,
+    jobs=[jobs.get_ecmwf_data],
```

Comment:
I know this looks weird, but locally all the other jobs still showed up in the UI? Might be worth you pulling and checking yourself. I'm not sure if it's because they're already initialised as ScheduledJobDefinitions, and as such, don't need to go in the `Definitions` object? But I could be wrong!

Reply:
Ah okay, yeah, that would be interesting. I am fine merging it for now, as I'm off tomorrow. But I'll check next week on it, it seems like it should have to be defined here, but yeah, not really sure.

Reply:
Yes it's definitely odd. The unit test checks that there are 18 jobs, not just one, and it is passing, so it does seem that they really are all defined still! But I'm certainly confused by it as well!